error[E0432]: unresolved import `std::sync::atomic::AtomicI64`
 --> .../staging_dir/target-*/host/share/cargo/registry/src/index.crates.io-*/pyo3-0.20.2/src/impl_/pymodule.rs:6:25
  |
6 | use std::sync::atomic::{AtomicI64, Ordering};
  |                         ^^^^^^^^^
  |                         |
  |                         no `AtomicI64` in `sync::atomic`
  |                         help: a similar name exists in the module: `AtomicI32`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `pyo3` (lib) due to previous error

--- a/src/rust/Cargo.toml
+++ b/src/rust/Cargo.toml
@@ -10,7 +10,7 @@ rust-version = "1.63.0"
 [dependencies]
 once_cell = "1"
 cfg-if = "1"
-pyo3 = { version = "0.20", features = ["abi3"] }
+pyo3 = { version = "<0.20", features = ["abi3"] }
 asn1 = { version = "0.15.5", default-features = false }
 cryptography-cffi = { path = "cryptography-cffi" }
 cryptography-key-parsing = { path = "cryptography-key-parsing" }
--- a/src/rust/cryptography-cffi/Cargo.toml
+++ b/src/rust/cryptography-cffi/Cargo.toml
@@ -8,7 +8,7 @@ publish = false
 rust-version = "1.63.0"
 
 [dependencies]
-pyo3 = { version = "0.20", features = ["abi3"] }
+pyo3 = { version = "<0.20", features = ["abi3"] }
 openssl-sys = "0.9.99"
 
 [build-dependencies]
--- a/src/rust/src/backend/ec.rs
+++ b/src/rust/src/backend/ec.rs
@@ -110,7 +110,7 @@ fn py_curve_from_curve<'p>(
     types::CURVE_TYPES
         .get(py)?
         .extract::<&pyo3::types::PyDict>()?
-        .get_item(name)?
+        .get_item(name)
         .ok_or_else(|| {
             CryptographyError::from(exceptions::UnsupportedAlgorithm::new_err((
                 format!("{name} is not a supported elliptic curve"),
